src/components/Authentication/__tests__/SuccessComponent.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SuccessComponent from '../Login/SuccessComponent';
+
+describe('SuccessComponentTest', () => {
+  it('render the component if success is truthy', () => {
+    const wrapper = shallow(<SuccessComponent success />);
```

Comment:
What is `success` and why are we passing it to the component?

Reply:
I had code complexity issues because of the number of ternary operators in the Login Component so I created a reusable component that I inherited from. success is the message that is going to be passed but I can see the component still gets to be tested if I omit that `success`

Reply:
Okay. Please remove it if it doesn't affect anything.

Reply:
Okey